src/SfxWeb/src/app/views/nodes/events/events.component.ts

```diff
@@ -12,14 +12,20 @@ export class EventsComponent implements OnInit {
 
   listEventStoreData: IEventStoreData<any, any> [];
 
-  constructor(public data: DataService) { }
+  constructor(public data: DataService, public settings: SettingsService) { }
 
   ngOnInit() {
-    this.listEventStoreData = [{
-      eventsList: this.data.createNodeEventList(null),
-      timelineGenerator: new NodeTimelineGenerator(),
-      displayName: 'Nodes'
-    }];
+    this.listEventStoreData = [
+      this.data.getNodeEventData(null)
```

Comment:
nit: remove the null, given its an optional param it would be better to just not put the null.